newton/examples/sensors/example_sensor_tiled_camera.py

```diff
@@ -167,6 +167,7 @@ def __init__(self, viewer: ViewerGL):
         self.camera_rays = self.tiled_camera_sensor.compute_pinhole_camera_rays(math.radians(fov))
         self.tiled_camera_sensor_color_image = self.tiled_camera_sensor.create_color_image_output()
         self.tiled_camera_sensor_depth_image = self.tiled_camera_sensor.create_depth_image_output()
+        self.tiled_camera_sensor_normal_image = self.tiled_camera_sensor.create_normal_image_output()
```

Comment:
Maybe for this example it would be nice to have an option to show the semantic image as well? The semantic indices could be just the shape index or some modulo thereof with a couple of predefined RGB colors to turn it into a human-viewable image.